### DISCUSSION TITLE
Change DNS server IP after install

### DISCUSSION THREAD

**Discussion by jaypfe** [Security-Onion-Solutions/securityonion#12263] [Unsupported Versions]
Hey guys! 

I am looking for a way to change the DNS IP I configured for a sensor when bringing it up. Is there anyway to do that? I was given the wrong IP for one of them, and I found out after the install was completed. 

Is this a supported configuration? Forgive me if this is in the docs, I went looking but I was unable to find it. 

Thanks!!

**Comment by cm-ops** [comment]
You could use nmcli to do that. `nmcli con mod $INT_NAME +ipv4.dns 8.8.8.8` to add and `nmcli con mod $INT_NAME -ipv4.dns 8.8.8.8` to remove.

**Reply by jaypfe** [reply]
That worked like a charm. Thank you!!

**Reply by TotieBash** [reply]
For future reference If you want little user friendly use nmtui.  It is the semi-gui version of nmcli.

**Reply by jaypfe** [reply]
ahh nice add! Thanks!
